tests/test_endpoints.py

```diff
@@ -383,6 +383,32 @@ def test_get_user(self):
         )
 
 
+class TestTranslations(TestCase):
+    def test_translations(self):
+        client = TestClient(APP)
+
+        response = client.get("/public/translations/")
+        self.assertEqual(response.status_code, 200)
+
+        data = response.json()
+
+        self.assertEqual(data["default_language_code"], "auto")
```

Comment:
If we set `default_language_code` to `"en"` we need to change this to `self.assertEqual(data["default_language_code"], "en")`